cql3/statements/alter_keyspace_statement.cc

```diff
@@ -111,3 +112,16 @@ cql3::statements::alter_keyspace_statement::prepare(database& db, cql_stats& sta
     return std::make_unique<prepared_statement>(make_shared<alter_keyspace_statement>(*this));
 }
 
+logging::logger cql3::statements::alter_keyspace_statement::_logger("alter_keyspace");
+
+future<::shared_ptr<cql_transport::messages::result_message>>
+cql3::statements::alter_keyspace_statement::execute(query_processor& qp, service::query_state& state, const query_options& options) const {
+    std::optional<sstring> warning = check_restricted_replication_strategy(qp.proxy(), keyspace(), *_attrs);
+    return schema_altering_statement::execute(qp, state, options).then([this,warning=std::move(warning)] (::shared_ptr<messages::result_message> msg) {
```

Comment:
code style: a couple of spaces are needed in the capture list: `[this,warning=std::move(warning)]` -> `[this, warning = std::move(warning)]`

Reply:
Ok. I'll need to do this in create_keyspace as well, as it has the same line.

Reply:
Done (both)